terraform/roar_variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "mix_validators" {
+  type    = bool
+  default = false
+  description = "Mix validator images for ROAR test"
```

Comment:
Is there value of having this variable ? can we simply change the existing  imag_version  to be a list of strings ?

Reply:
This makes it so that we're backwards compatible with some of the older terraform configs. Also figured that mixing validator versions would only be valuable in the case of ROAR testing, so use a separate switch variable